solutions/standard/main.tf

```diff
@@ -148,3 +148,60 @@ data "ibm_resource_instance" "existing_sm" {
   count      = var.existing_secrets_manager_crn == null ? 0 : 1
   identifier = var.existing_secrets_manager_crn
 }
+
+#######################################################################################################################
+# Secrets Manager Event Notifications Configuration
+#######################################################################################################################
+
+locals {
+  parsed_existing_en_instance_crn = var.existing_event_notification_instance_crn != null ? split(":", var.existing_kms_instance_crn) : []
+  en_region                       = length(local.parsed_existing_en_instance_crn) > 0 ? local.parsed_existing_kms_instance_crn[5] : null
+  existing_en_guid                = length(local.parsed_existing_en_instance_crn) > 0 ? local.parsed_existing_kms_instance_crn[7] : null
+}
+
+data "ibm_en_destinations" "en_destinations" {
+  count         = var.existing_event_notification_instance_crn != null ? 1 : 0
+  instance_guid = local.existing_en_guid
+}
+
+data "ibm_iam_users" "users_profiles" {
+}
+
+resource "ibm_en_topic" "en_topic" {
+  count         = var.existing_event_notification_instance_crn != null ? 1 : 0
+  instance_guid = local.existing_en_guid
+  name          = "Secrets Manager Topic"
+  description   = "Topic for Secrets Manager events routing"
+  sources {
+    id = local.secrets_manager_crn
+    rules {
+      enabled           = true
+      event_type_filter = "$.*"
+    }
+  }
+}
+
+resource "ibm_en_subscription_email" "email_subscription" {
+  count          = var.existing_event_notification_instance_crn != null ? 1 : 0
+  instance_guid  = var.existing_event_notification_instance_crn
```

Comment:
again, should we support creating email subscription in existing EN instance?

Reply:
if the existing_event_notifcation_instance_crn is not `null`, the resource is created, else don't create the resource.

Reply:
yea my bad - I was thinking we supported creating EN in this Da - ignore my comment